apps/video/test/marionette/lib/video.js

```diff
@@ -130,14 +133,30 @@ Video.prototype = {
    * @return {Marionette.Element} First element of all thumbnail images.
    */
   get thumbnail() {
-    return this.client.findElement(Video.Selector.thumbnail);
+    return this.client.helper.waitForElement(Video.Selector.thumbnail);
   },
 
   /**
    * @return {Marionette.Element} video player element.
    */
   get player() {
-    return this.client.findElement(Video.Selector.player);
+    return this.client.helper.waitForElement(Video.Selector.player);
+  },
+
+  showControls: function() {
+    this.player.tap();
```

Comment:
Should you do a waitForElement() here to make sure that the controls are visible before you return?  That would only make sense if you were able to first make sure that they were not already visible before you called tap.